.github/workflows/build-standalone.yml

```diff
@@ -0,0 +1,74 @@
+on:
+  release:
+    types: [created]
+
+name: Build Standalone CLI
+
+jobs:
+  build_cli:
+    name: Build CLI on MacOS
+    runs-on: macos-latest
+    env:
+      APPLE_APP_STORE_CONNECT_KEY_ID: ${{ secrets.APPLE_APP_STORE_CONNECT_KEY_ID }}
+      APPLE_APP_STORE_CONNECT_ISSUER_ID: ${{ secrets.APPLE_APP_STORE_CONNECT_ISSUER_ID }}
+      APPLE_CODESIGN_CERT_ID: "F9F7EEE981D7A60F83EBEC36AD675448CA0AB41B"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ^22
+          cache: yarn
+          cache-dependency-path: |
+            yarn.lock
```

Comment:
Use `actions/setup-node@v4`, it would also be best to use `node-version-file: '.nvmrc'` instead of `node-version: ^22`. You'll need to create the `.nvmrc` file, this can be done with `nvm use [<version>] --save`.